src/router/index.js

```diff
@@ -143,7 +143,14 @@ function createServerlessApp() {
         '/results',
         asyncMiddleware(async (req, res) => {
             let { d } = req.query
-            const unpacked = jsonpack.unpack(d)
```

Comment:
I believe we would like to keep both implementations here in order to fallback to `jsonpack` if the data isn't compressed so that older links are still accessible, right?

Reply:
That would probably be ideal, yes. I'll have to find some time to get back to this but some possibilities might be:

* Try to detect if the data has been compressed with `lz-string` in the handler here, although I don't see provided function from `lz-string` to do this.
* Create a new major version with this breaking change.
* Add a new request param in the CLI/Node app [here](https://github.com/bundlewatch/bundlewatch/pull/436/files#diff-00631608455fa08c21e5654ec10b8b0221c082640f8fc03672591f4925ec1313R33) that indicates compression with `lz-string`, then check for the presence of that request param in this handler.

My preference would be for the last option.